assets/js/18f-guide.js

```diff
@@ -0,0 +1,11 @@
+$(function() {
+  $('.sidebar-nav').each(function() {
+    accordion($(this));
+  });
+});
+
+// http://bryanbraun.github.io/anchorjs/
+anchors.options = {
+  visible: 'touch'
+};
+anchors.add('h3,h4,h5');
```

Comment:
It didn't seem that useful to link to the page title, which is an `<h2>`.

https://github.com/18F/guides-style/blob/51a8eb38d6397b076a697607fd2d8aae5e39cd58/lib/guides_style_18f/layouts/default.html#L26

My thinking was that all of the sub-sections below should be `<h3>`s, `<h4>`s, etc., but I don't feel super strongly about it.